app/src/main/java/com/easyrouteapp/component/SlidingTabStrip.java

```diff
@@ -0,0 +1,155 @@
+package com.easyrouteapp.component;
+
+/**
+ * Created by viniciusthiengo on 5/18/15.
+ */
+import android.R;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.util.AttributeSet;
+import android.util.TypedValue;
+import android.view.View;
+import android.widget.LinearLayout;
+
+class SlidingTabStrip extends LinearLayout {
```

Comment:
please, if it's not a library, give the credit to who created the class you are using.